conf/application.conf

```diff
@@ -60,6 +62,16 @@ auditing {
   enabled = true
 }
 
+mongodb {
+  uri = "mongodb://localhost:27017/"${appName}
+  encryption.key = "QmFyMTIzNDVCYXIxMjM0NQ=="
+  timeToLive = 15minutes
```

Comment:
This isn't needed here. It only exists in state benefits because that service has a second mongo repository with a different TTL defined.